waku/v2/node/waku_node.nim

```diff
@@ -258,10 +263,32 @@ proc new*(T: type WakuNode,
     peerManager: PeerManager.new(switch, peerStorage),
     switch: switch,
     rng: rng,
-    enr: enr,
+    enr: populatedEnr,
```

Comment:
Provide constructed `wakuDiscv5` as an argument here.

Reply:
Addressed in [74c3622](https://github.com/waku-org/nwaku/pull/1512/commits/74c36223864222f0da14cee19f4a8f9b1f7f3654)